### PR TITLE
Feaure/cp3 638

### PR DESCRIPTION
**Pull request by ocampd1**

### CODE REVIEW

File: chucks-pick3-mobile/gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  // preserving the folder structure
+  console.log("inside inlcude-mocks gulp tasks.");
+  if (process.env.USE_MOCKS === "true") {
+    src(["test/mocks/**/*.*", "test/environments/**/*.*"]).pipe(dest("src"));
+  }
+
+  done();
+});
+
+task("remove-prod", function () {
```
Comment (pouncilt) [inline chucks-pick3-mobile/gulpfile.js:20]: Let's remove this method.  We don't need it.

File: chucks-pick3-mobile/gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  return src(["src/**/*prod*.*"], { read: false }).pipe(clean());
+});
+task("include-prod", function (done) {
```
Comment (pouncilt) [inline chucks-pick3-mobile/gulpfile.js:23]: Let's remove this method.  We don't need it.

File: chucks-pick3-mobile/gulpfile.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  // the base option sets the relative root for the set of files,
+  // preserving the folder structure
+  console.log("inside inlcude-mocks gulp tasks.");
+  if (process.env.USE_MOCKS === "true") {
+    src(["test/prod/**/*.*", "test/environments/**/*.*"]).pipe(dest("src"));
+  }
+
```
Comment (pouncilt) [inline chucks-pick3-mobile/gulpfile.js:30]: We need to create a gulp task that run the ng serve command and dynamically adds the `--configuration=mock` parameter if `process.env.USE_MOCKS === "true"`

It should do the same thing as original start script: `" ionic serve --configuration=mock"`

See if you can tell gulp task to run the `" ionic serve --configuration=mock"` command.

File: chucks-pick3-mobile/package.json [hunk 4]
```diff
@@ -5,19 +5,20 @@
   "homepage": "https://ionicframework.com/",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve",
-    "build": "ng build",
-    "test": "ng test",
+    "serve": "npm run use-mocks && ionic serve --configuration=mock",
```
Comment (pouncilt) [inline chucks-pick3-mobile/package.json:8]: Remove serve command

File: chucks-pick3-mobile/package.json [hunk 5]
```diff
@@ -5,19 +5,20 @@
...
+    "start": "npm run use-prod && ionic serve --configuration=prod",
```
Comment (pouncilt) [inline chucks-pick3-mobile/package.json:9]: Start script needs to call gulp task to start server; so revert this script and start again.  For example, we should have something like this: `npm run use-mock && npm run server`

**Review by pouncilt — CHANGES_REQUESTED** [review]

File: chucks-pick3-mobile/gulpfile.js (hunk 3 above)
Comment (ocampd1) [reply chucks-pick3-mobile/gulpfile.js:30]: Completed work on gulp task.

File: chucks-pick3-mobile/gulpfile.js (hunk 1 above)
Comment (ocampd1) [reply chucks-pick3-mobile/gulpfile.js]: Resolved

File: chucks-pick3-mobile/gulpfile.js (hunk 2 above)
Comment (ocampd1) [reply chucks-pick3-mobile/gulpfile.js:30]: Resolved

**Review by pouncilt — APPROVED** [review]

**Pull request merged**